library/ecp.c

```diff
@@ -2333,6 +2383,10 @@ int mbedtls_ecp_mul( mbedtls_ecp_group *grp, mbedtls_ecp_point *R,
              const mbedtls_mpi *m, const mbedtls_ecp_point *P,
              int (*f_rng)(void *, unsigned char *, size_t), void *p_rng )
 {
+    ECP_VALIDATE_RET( grp != NULL );
```

Comment:
Not a blocker: `mbedtls_ecp_mul_restartable` will do the exact same checks, so the ones here are redundant. Also applies to other functions that are trivial wrappers around some other function (`xxx` → `xxx_restartable`, `mbedtls_ecp_gen_keypair` → `mbedtls_ecp_gen_keypair_base` except for `grp`, `mbedtls_ecp_gen_keypair_base` → `mbedtls_ecp_gen_privkey`).